### ISSUE TITLE
Balancing the game futher

### ISSUE THREAD

**Issue by akien-mga** [KOBUGE-Games/project-kumquat#33]
Some ideas to balance the game further:
- Make enemies spawn less frequently, so that you don't have to fight 200 of them at a given time
- Make the enemies stronger, to counter balance their lower number

Further ideas welcome :)

**Comment by est31** [comment]
Yeah, 200 enemies aren't just bad from a balance standpoint, they also cause a huge FPS lag.

**Comment by ObaniGemini** [comment]
:+1: @est31 . Maybe should we rarify spawns also ?
